<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1213</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="8,2">
      <item>
       <widget class="QLineEdit" name="url_Input">
        <property name="text">
         <string>http://sso.ksedu.cn/SignIn.aspx</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="go">
        <property name="text">
         <string>go</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="8,3">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>view</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QFrame" name="frame_view">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>Import</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QFrame" name="frame_setting">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>351</width>
               <height>531</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPlainTextEdit" name="plainTextEdit_users">
                <property name="plainText">
                 <string>QIANYT19
</string>
                </property>
                <property name="placeholderText">
                 <string>Import accounts in batches, one per line</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPlainTextEdit" name="plainTextEdit_pwds">
                <property name="plainText">
                 <string></string>
                </property>
                <property name="placeholderText">
                 <string>Import password in batches, one per line</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="geometry">
              <rect>
               <x>420</x>
               <y>40</y>
               <width>311</width>
               <height>141</height>
              </rect>
             </property>
             <property name="title">
              <string>Import</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QCheckBox" name="checkBox">
                <property name="text">
                 <string>Use same password</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_3">
                <property name="text">
                 <string>Import user</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_6">
                <property name="text">
                 <string>Import password</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QPlainTextEdit" name="plainTextEdit_pwds_2">
             <property name="geometry">
              <rect>
               <x>420</x>
               <y>250</y>
               <width>401</width>
               <height>291</height>
              </rect>
             </property>
             <property name="plainText">
              <string>window.open('/Space/StudentSkipPage');</string>
             </property>
             <property name="placeholderText">
              <string>Automated execution</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit">
             <property name="geometry">
              <rect>
               <x>420</x>
               <y>200</y>
               <width>381</width>
               <height>21</height>
              </rect>
             </property>
             <property name="text">
              <string>http://sso.ksedu.cn/Space/StudentSkipPage</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_oper">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QGroupBox" name="groupBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>291</width>
           <height>211</height>
          </rect>
         </property>
         <property name="title">
          <string>GroupBox</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="7" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>password</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>next</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="2">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>uid</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="pushButton_5">
            <property name="text">
             <string>login in</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>previous</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="lineEdit_code"/>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QSpinBox" name="spinBox">
            <property name="maximum">
             <number>65535</number>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QPushButton" name="pushButton_9">
            <property name="text">
             <string>runjs</string>
            </property>
           </widget>
          </item>
          <item row="8" column="2">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>410</y>
           <width>291</width>
           <height>151</height>
          </rect>
         </property>
         <property name="title">
          <string>GroupBox</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_4">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>320</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>back</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_7">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>320</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>forward</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_8">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>320</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>reload</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1213</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
